planner/core/prepare_test.go

```diff
@@ -1107,7 +1149,7 @@ func (s *testPlanSerialSuite) TestPlanCachePointGetAndTableDual(c *C) {
 	tk.MustQuery("select @@last_plan_from_cache").Check(testkit.Rows("0"))
 	// Must not reuse the previous TableDual plan.
```

Comment:
Please modify the comment here. Besides, we need to check the plan from the cache.